src/components/installer/installer.ts

```diff
@@ -0,0 +1,187 @@
+'use strict';
+
+import * as download from 'download';
+import * as fs from 'fs';
+import mkdirp = require('mkdirp');
+import * as path from 'path';
+import * as tar from 'tar';
+import * as tmp from 'tmp';
+import * as vscode from 'vscode';
+import { Shell, Platform } from '../../shell';
+import { Errorable } from '../../wizard';
+import { exec } from 'child_process';
+
+export async function installKubectl(shell: Shell) : Promise<Errorable<void>> {
+    const tool = 'kubectl';
+    const installFolder = getInstallFolder(shell, tool);
+    mkdirp.sync(installFolder);
+    switch (shell.platform()) {
+        case Platform.Windows:
+        {
+            // Instruction say always to use this link for Windows, not to check the version first
+            const kubectlUrl = 'https://storage.googleapis.com/kubernetes-release/release/v1.9.6/bin/windows/amd64/kubectl.exe';
```

Comment:
See the document. https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl

The template `https://storage.googleapis.com/kubernetes-release/release/${version}/bin/${os}/amd64/kubectl.exe`  is also applied to windows. I think we need do that like what linux and mac did.

Reply:
Darn, you're right.  Thanks for spotting!